src/resources/js/Layouts/AuthenticatedLayout.jsx

```diff
@@ -27,6 +27,12 @@ export default function Authenticated({ user, header, children }) {
                                 <NavLink href={route('posts.index')} active={route().current('posts.index')}>
                                     Posts
                                 </NavLink>
+                                <NavLink href={route('filter')} active={route().current('filter')}>
```

Comment:
I'd say only an admin can add labels and types, right?

Reply:
I will implement that yes, I wanted to have what I already have merged into main already though. Before my branch becomes to distant.